sounddevice.py

```diff
@@ -1893,9 +1903,19 @@ def input_overflow(self):
         This can happen in full-duplex and input-only streams (including
         `playrec()` and `rec()`).
 
+        The bit may be set with ``input_overflow = True``, and
+        cleared with ``input_overflow = False``.
+
         """
         return self._hasflag(_lib.paInputOverflow)
 
+    @input_overflow.setter
+    def input_overflow(self, new_value):
+        if new_value:
+            self._setflag(_lib.paInputOverflow)
+        else:
+            self._clearflag(_lib.paInputOverflow)
```

Comment:
I guess it doesn't make a huge difference, but you are repeating the `if` statement for each flag.

What about moving it to the helper function?

```python
    @input_overflow.setter
    def input_overflow(self, value):
        self._updateflag(_lib.paInputOverflow, value)

    ...

    def _updateflag(self, flag, value):
        if value:
            self._flags |= flag
        else:
            self._flags &= ~flag
```

Reply:
I feel embarrassed I didn't see this myself!  Thanks :o)

Reply:
No need to be embarrassed.

That's what reviews are for. It's always good to have a second (or third etc.) set of eyes looking at the code.